common/changes/@itwin/analytical-backend/dan-3.7.4-finalize-release_2023-04-26-18-31.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "type": "none",
+      "packageName": "@itwin/analytical-backend"
+    }
+  ],
+  "packageName": "@itwin/analytical-backend",
+  "email": "Daniel.Rodriguez@bentley.com"
```

Comment:
why is your internal/private bentley email being exposed?
the rush.json at the root forces you to use your gh email